client/test/account_store.spec.js

```diff
@@ -396,8 +396,26 @@ describe('AccountStore', () => {
 
 
     describe('Should REMOVE account(s)', () => {
-      it.skip('REMOVE_ACCOUNT_SUCCESS', () => {
-        // FIXME Nothing is done here in the store code.
+      beforeEach(() => {
```

Comment:
RESET_ACCOUNT is a test-specific action and should be removed. I think the proper way to test is to start from a clean state (it's what the freshRequire(store) in specs_dispatcher is for). 
Or to test reducers as unit.
To be discussed.